app/services/wrappers/geocoder.rb

```diff
@@ -29,7 +27,7 @@ def site_options(address_string = nil, coordinates = nil)
     site_params[:addressString] = address_string if address_string.present?
     site_params[:parcelPoint] = coordinates.join(",") if coordinates.present?
 
-    r = get("/addresses.#{OUTPUT_FORMAT}", site_params)
```

Comment:
I don't see a reason the output format variable needs to refactored?

Reply:
its just unnecessary as we have established will always use the JSON output format. Should I put this back?